src/Logic.hs

```diff
@@ -251,57 +251,63 @@ updateStateVar var state = void $ atomically $ swapTMVar var state
 readStateVar :: StateVar -> IO ProjectState
 readStateVar var = atomically $ readTMVar var
 
+clearPullRequest :: PullRequestId -> PullRequest -> ProjectState -> Action ProjectState
```

Comment:
Can we have a test of that wrt to changing the base branch? So basically that something like `[CommentAdded prId "deckard" "@bot merge", PullRequestEdited prId "blah blah" (Branch "master"), CommentAdded prId "deckard" "@bot merge" ]` results in the PR **not** rejected?